include/test/battle.h

```diff
@@ -199,6 +199,17 @@
  * - Instead of player and opponent there is playerLeft, playerRight,
  *   opponentLeft, and opponentRight.
  *
+ * AI_BATTLE_TEST(name, results...) and AI_DOUBLE_BATTLE_TEST(name, results...)
+ * Define battles where opponent mons are controlled by AI, the same that runs
+ * when battling regular Trainers. The flags for AI should be specified by
+ * the AI_FLAGS command.
+ * The rules remain the same as with the SINGLE and DOUBLE battle tests
+ * with some differences:
+ * - opponent's action is specified by the EXPECTED_MOVE(s) / EXPECTED_SEND_OUT / EXPECTED_SWITCH commands
```

Comment:
I think I'd call these `EXPECT_MOVE`(s), `EXPECT_SEND_OUT`, and `EXPECT_SWITCH` for consistency with `EXPECT`/`EXPECT_EQ`/`EXPECT_FASTER`/etc.